src/platform/os_ipc_stub.c

```diff
@@ -3,7 +3,11 @@
 #include <string.h>
 #include <errno.h>
 #include <unistd.h>
+
+#ifndef __FreeBSD__
```

Comment:
Whoops, too late!  Hopefully it won't be a problem or it'll be clear to the next person what to fix.